src/components/layout/header.js

```diff
@@ -68,29 +70,51 @@ const Menu = () => {
   ]
 
   return (
-    <Box component="nav" sx={{
-      flex: 1,
-      display: 'flex',
-      justifyContent: 'center',
-    }}>
-      {
-        mainMenuLinks.map((page) => (
-          <Link
-            to={`/${page.slug}`}
-            key={ `main-menu-${ page.slug }` }
-            activeClassName="active"
-          >
-            { page.title }
-          </Link>
-        ))
-      }
-    </Box>
+    <Fragment>
+      <Box component="nav" sx={{
+        flex: 1,
+        display: 'flex',
+        justifyContent: 'center',
+      }}>
+        {
+          mainMenuLinks.map((page) => (
+            <Link
+              to={`/${page.slug}`}
+              key={ `main-menu-${ page.slug }` }
+              activeClassName="active"
+            >
+              { page.title }
+            </Link>
+          ))
+        }
+      </Box>
+      <Button
```

Comment:
This `mailto` link doesn't work for me. Looks like swapping the `to` attribute with `href` works.

Reply:
might also want to add `target="_blank"` to open the mail app in a new tab